app/code/Magento/Catalog/Controller/Adminhtml/Category/Edit.php

```diff
@@ -72,6 +80,8 @@ public function execute()
             $this->getRequest()->setParam('id', $categoryId);
         }
 
+        // TODO: Ensure this feature is enabled in store config before running this
+        $this->loadCategoryAllScopeData();
```

Comment:
This method will cause an ExtensionPool ReadHandler to get added for category objects. This will result in an extension attribute being added to the category that shows store view specific data. Example:

![hackathon2017 dev -ykmky](https://cloud.githubusercontent.com/assets/129031/24584645/e08a2d8a-1728-11e7-89ac-b684dc442d19.png)